src/main/java/seedu/address/model/person/EditPersonDescriptor.java

```diff
@@ -0,0 +1,226 @@
+package seedu.address.model.person;
```

Comment:
What is the reason to separate out the EditPersonDescriptor from EditCommand?

Reply:
So that it can be reused in the DeleteFieldCommand, without referencing from another command class. This is because deleting fields is essentially editing a person.